src/wallet/rpcwallet.cpp

```diff
@@ -1188,6 +1187,7 @@ UniValue addmultisigaddress(const JSONRPCRequest& request)
             "       ...,\n"
             "     ]\n"
             "3. \"account\"      (string, optional) DEPRECATED. An account to assign the addresses to.\n"
+            "4. \"address_type\" (string, optional) The address type to use. Options are \"legacy\", \"p2sh-segwit\", and \"bech32\". Default is set by -addresstype.\n"
```

Comment:
You need to update the RPC command table as well with this added argument, to make named arguments work.

Reply:
Maybe even add a test for all new options

Reply:
@promag you missed this update to the RPC command table for `addmultisigaddress`